src/mkdocs_spellcheck/plugin.py

```diff
@@ -89,7 +91,10 @@ def on_config(self, config: MkDocsConfig) -> MkDocsConfig | None:
         """
         self.strict_only = self.config["strict_only"]
         self.backends_config = self.config["backends"]
-        self.skip_files = self.config["skip_files"]
+        # Convert the skip files into a set of Path objects to make it
+        # OS-agnostic and increase lookup efficiency
+        self.skip_files = {Path(x) for x in self.config["skip_files"]}
```

Comment:
Could you move this change to another PR, so that we have two different fix and feat commits?

Reply:
I removed this change, but the changes to add fnmatch still fix the issue, so the new feature also fixes the bug.